data/json/npcs/godco/NPC_Helena_Misinter.json

```diff
@@ -0,0 +1,280 @@
+          ]
+        },
+        "topic": "TALK_GODCO_Helena_Contact_FreeMerch"
+      },
+      { "text": "Well, it's been nice talking to you, but I need to head out.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_GODCO_Helena_Story",
+    "dynamic_line": "I am a prophet, like my uncle before me.  He saw <the_cataclysm>, the righteous judgement of God and set about preparing an ark to save us.  He passed before his visions could be fulfilled, and now I carry his righteous banner.",
+    "responses": [
+      { "text": "Tell me about his visions.", "topic": "TALK_GODCO_Helena_Uncle" },
+      { "text": "Let's talk about something else.", "topic": "TALK_GODCO_Helena_2" },
+      { "text": "Well, it's been nice talking to you, but I need to head out.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_GODCO_Helena_Uncle",
+    "dynamic_line": "He had many visions, and he had many secrets.  I don't know all of them, but there was one he was highly vocal about, regarding <the_cataclysm>, that according to him \"The spawn of Satan shall descend upon the Earth, and all will be consumed in death and destruction\".  He never said much else about it, and always had a mortified face whenever someone brought it up.",
+    "responses": [
+      { "text": "Let's talk about something else.", "topic": "TALK_GODCO_Helena_2" },
+      { "text": "Well, it's been nice talking to you, but I need to head out.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_GODCO_Helena_Mood",
+    "dynamic_line": "Thanks to my uncle's forethought, we've been blessed by God.  We have plenty of food for ourselves, safety from the hordes of Hell, and the continued protection of the Lord our God.",
+    "responses": [
+      { "text": "Let's talk about something else.", "topic": "TALK_GODCO_Helena_2" },
+      { "text": "Well, it's been nice talking to you, but I need to head out.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_GODCO_Helena_Community",
+    "dynamic_line": "They denied us, they labeled us as a doomsday cult, threw sticks and stones at us, and yet, we were standing resiliently.  Why?  Because we knew God was on our side.",
+    "responses": [
+      { "text": "Let's talk about something else.", "topic": "TALK_GODCO_Helena_2" },
+      { "text": "Well, it's been nice talking to you, but I need to head out.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_GODCO_Helena_Join",
+    "dynamic_line": "We are all under God's protection here.  Our community, like all moral civilizations, has a set of rules.  Obey the Decalogue, have respect for your neighbors and we'll get along just fine.  Go see Gemma or Julian for a bed, if one's available.  We do have limited space on our Ark, though.",
```

Comment:
I'd like to see these requests to join rebuffed if you have joined the Norse Pagans or are wearing their symbol.

Reply:
The Norse Evangelicals don't really have a symbol per se. If you were wearing one of the wolf skull helmets, Helena would probably assume you either killed a "demon" or found someone's old Halloween costume. Outside of a symbol, Helena has no way of knowing you associated with them unless you tell her. Even if you did, Helena wouldn't know who they are, and the conversation would probably wind up with the player bargaining information in exchange for joining. I doubt anyone would be keen on telling Helena, considering her extreme views. 
If she somehow found out after you joined, depending on your trust with the community, she probably wouldn't care and see you as a "sinner saved by grace". 

With the hordes of zombies roaming the countryside and the general lack of basic supplies, the New England Church Community and the Norse Evangelicals likely wouldn't care about each other unless one goes on the offensive. The leadership of both factions is smart enough to realize a war between the two would be a waste of time, manpower and materials.